src/webvr-manager.js

```diff
@@ -98,9 +98,10 @@ function WebVRManager(renderer, effect, params) {
       this.isVRCompatible = true;
     } else if (hmd) {
       this.isVRCompatible = true;
+      this.usingPolyfill = hmd.deviceName.indexOf('webvr-polyfill') === 0;
       // Only enable distortion if we are dealing using the polyfill, we have a
```

Comment:
webvr-1.0 doesn't have this problem because it doesn't use requestFullscreen to present to the VR display. The good news is that webvr-1.0 works in stable Firefox for Android. But it breaks in Nightly, since Nightly is still on the old API (and will be for months, AFAIK). @cvan?